### ISSUE TITLE
Error when installing: Server is not defined! -> usbip.conf seems to be empty

### ISSUE THREAD

**Issue by Zaptro** [barbalion/usbip-win-client#5]
Hello,

as said in the topic the install.cmd throws me the error message:

Error: Server is not defined! Run 'install.cmd' first.
Error:   "" is disconnected.

This is despite me specifiyng the server and the second time running install.cmd correctly recalling, that I already made a config and showing me the correct server ip.

I hope you can help, I much appreciate how much effort must have went into writing all these scripts.

Yours sincerely
Zaptro

**Comment by Zaptro** [comment]
It seems indeed that the "move config_new to config"-command seems to not write the usbip.conf properly. Sadly the script autodisconnects the device immediately after connecting with the following output:

- 192.168.178.25
    1-1.1.3: ROCCAT : Sova Keyboard (1e7d:2f76)
           : /sys/devices/platform/scb/fd500000.pcie/pci0000:00/0000:00:00.0/0000:01:00.0/usb1/1-1/1-1.1/1-1.1.3
           : (Defined at Interface level) (00/00/00)
           :  0 - Human Interface Device / Boot Interface Subclass / Keyboard (03/01/01)
           :  1 - Human Interface Device / No Subclass / None (03/00/00)
           :  2 - Human Interface Device / No Subclass / None (03/00/00)

Error: 192.168.178.25  "" is disconnected.
WARNING: It looks not working :(
Try to install the certificate and the driver manually.

Connecting a device is only possible through manually doing the command "usbip attach_ude -r 192.168.178.25 -b 1-1.1.1". But somehow I cannot connect multiple devices? Trying to open another cmd-terminal and using e.g. "usbip attach_ude -r 192.168.178.25 -b 1-1.1.3" gives me error 0xb7. Is the windows driver not able to attach multiple devices?

**Comment by barbalion** [comment]
Hey!
I've made a few stability fixes. Please try to install it again.
To address your driver issue try to install non-ude driver (answer 'no' to "use ude driver" during the installation process).

**Comment by Zaptro** [comment]
Hey,
wow thank you so much for looking into this issue so fast and even provide a revised version of the scripts so fast!! :-) The install.cmd now works flawless and the non-ude driver also works better for me.

I also searched a little bit more into why the ude driver makes this problem and I found this post, that decribes a possible solution: [attach_ude multiple clients?](https://github.com/cezanne/usbip-win/issues/191). Maybe this is helpful to you or someone else that somehow wants to use the ude driver with multiple devices.

Greetings
Zaptro
